### PR TITLE
Fix kubeconform error msg

### PR DESCRIPTION
**Pull request by spiarh**

### CODE REVIEW

**Review by prit342 — DISMISSED** [review]
LGTM

File: test/1_cli.bats [hunk 1]
```diff
@@ -217,24 +216,24 @@ teardown() {
...
   CRITICAL_SECOND_SECOND=$(jq -r .[1].scoring.critical[1].points <<<"${output}")
-  (( CRITICAL_SECOND_FIRST <= CRITICAL_SECOND_SECOND ))
+  ((CRITICAL_SECOND_FIRST <= CRITICAL_SECOND_SECOND))
 
   # report 2 - advisories - magnitude sort/highest number first
   ADVISE_SECOND_FIRST=$(jq -r .[1].scoring.advise[0].points <<<"${output}")
   ADVISE_SECOND_SECOND=$(jq -r .[1].scoring.advise[1].points <<<"${output}")
   ADVISE_SECOND_THIRD=$(jq -r .[1].scoring.advise[2].points <<<"${output}")
-  (( ADVISE_SECOND_FIRST >= ADVISE_SECOND_SECOND >= ADVISE_SECOND_THIRD ))
+  ((ADVISE_SECOND_FIRST >= ADVISE_SECOND_SECOND >= ADVISE_SECOND_THIRD))
```
Comment (06kellyjac) [inline test/1_cli.bats:236]: The bash styleguide we follow has spaces after `((` and before `))`

https://google.github.io/styleguide/shellguide.html#arithmetic

**Review by 06kellyjac — CHANGES_REQUESTED** [review]
looks good, minor styling nit

File: test/1_cli.bats (hunk 1 above)
Comment (spiarh) [reply test/1_cli.bats]: @06kellyjac updated, FI, this is the result of `shfmt`

Comment (spiarh) [reply test/1_cli.bats]: $(( and (( ambiguity is not supported. Backtracking would complicate the parser and make streaming support via io.Reader impossible. The POSIX spec recommends to [space the operands](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03) if $( ( is meant.
$ echo '$((foo); (bar))' | shfmt
1:1: reached ) without matching $(( with ))

**Review by 06kellyjac — APPROVED** [review]
LGTM, thanks @spiarh :rocket:

**Pull request merged**